### PR TITLE
Practice2

### PR DESCRIPTION
**Pull request by andrew-budnitskiy**

### CODE REVIEW

**Comment by vasilevkin** [comment]
create a new repo for Practice 2

File: Cats/Cats.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -0,0 +1,414 @@
+// !$*UTF8*$!
```
Comment (vasilevkin) [inline Cats/Cats.xcodeproj/project.pbxproj:1]: do not include this file to the pr

File: Cats/Cats.xcodeproj/project.xcworkspace/contents.xcworkspacedata [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (vasilevkin) [inline Cats/Cats.xcodeproj/project.xcworkspace/contents.xcworkspacedata:1]: do not include this file to the pr

File: Cats/Cats.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (vasilevkin) [inline Cats/Cats.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:1]: do not include this file to the pr

File: Cats/Cats/AppDelegate.swift [hunk 4]
```diff
@@ -0,0 +1,36 @@
+//
```
Comment (vasilevkin) [inline Cats/Cats/AppDelegate.swift:1]: do not include this file to the pr

File: Cats/Cats/Info.plist [hunk 5]
```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (vasilevkin) [inline Cats/Cats/Info.plist:1]: do not include this file to the pr

File: Cats/Cats/SceneDelegate.swift [hunk 6]
```diff
@@ -0,0 +1,52 @@
+//
```
Comment (vasilevkin) [inline Cats/Cats/SceneDelegate.swift:1]: do not include this file to the pr

**Review by vasilevkin — CHANGES_REQUESTED** [review]
only your changes should be in the pr

**Comment by vasilevkin** [comment]
please do not close pr.
make it draft and edit.

**Comment by andrew-budnitskiy** [comment]
According your comment I create new repo and I’m waiting for your collaborator’s confirm now.

Отправлено с iPhone

**Pull request closed without merging**